faces-tck/run-tck.sh

```diff
@@ -3,10 +3,20 @@ if [ JAVA_HOME = "" ] ; then
     export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64
 fi
 
-rm -r target
-mkdir target
-wget -q https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee10/staged/eftl/jakarta-faces-tck-4.0.1.zip -O target/jakarta-faces-tck-4.0.1.zip
+#rm -r target
+#mkdir target
```

Comment:
I'm pretty certain you can't leave these commented out.
See me previous review - it fails to download to `target/jakarta-faces-tck-4.0.1.zip` because the `target` dir doesn't exist.

Reply:
You are right, I was running it locally with some previous downloads and commented it out to speed up tests. Fixing...

Reply:
fixed